APIs/SinkMetadataProcessingAPI.raml

```diff
@@ -47,52 +47,58 @@ mediaType: application/json
               type: !include schemas/resource-list.json
           404:
             description: Returned when the requested Sender ID does not exist.
-    /media-profiles:
+    /constraints:
       get:
-        description: Show the last media profile array put before.
+        description: Show the last applied Constraints.
         responses:
           200:
             body:
-              example: !include ../examples/media-profiles-get-200.json
-              type: !include schemas/media_profiles.json
+              example: !include ../examples/constraints-get-200.json
+              type: !include schemas/constraints.json
           205:
-            description: Denotes an external change occurred which violated the previous Media Profiles.
+            description: Denotes an external change occurred which led to the current Constraints violation.
```

Comment:
Should we use the status endpoint instead of status code here to signal external conditions?

Reply:
2020/11/01 activity group call: covered by another comment in this PR. Marked as resolved.